app/src/main/java/com/github/sdp/ratemyepfl/model/items/Classroom.kt

```diff
@@ -10,21 +9,20 @@ data class Classroom(
     override var numRatings: Int = 0,
     override var avgRating: Double = 0.0,
     val type: String? = null,
-    val reviews: List<ClassroomReview>? = null
 ) : Reviewable()
 {
     val name: String = type.orEmpty()
 
-    constructor(id: String, name: String, reviews: List<ClassroomReview> = listOf())
-        : this(id, 0, 0.0, name, reviews)
+    constructor(id: String, name: String)
+        : this(id, 0, 0.0, name)
 
     // This converts a json object from firebase into a Classroom object
     companion object {
         fun DocumentSnapshot.toClassroom() : Classroom? {
             return try {
                 //val id = getString("id")!!
                 //val name = getString("name")!!
-                Classroom(id, "",  listOf())
```

Comment:
Maybe we could make a constant value for "" in a utils/ object.

Reply:
In my opinion, it will just add complexity, having to go to utils/object just to understand what the parameter is. Especially for a short one like "".